packages/victory-chart/src/victory-chart.js

```diff
@@ -87,10 +87,18 @@ export default class VictoryChart extends React.Component {
   getNewChildren(props, childComponents, calculatedProps) {
     const children = getChildren(props, childComponents, calculatedProps);
     const getAnimationProps = Wrapper.getAnimationProps.bind(this);
-    return children.map((child, index) => {
+    const backgroundComponent = getBackgroundWithProps(props, calculatedProps);
```

Comment:
Please move this calculation into the `props.style.background` conditional so we can skip it if we don't need it.